controllers/nginx/rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -206,7 +206,8 @@ http {
         {{ if eq $upstream.SessionAffinity.AffinityType "cookie" }}
         sticky hash={{$upstream.SessionAffinity.CookieSessionAffinity.Hash}} name={{$upstream.SessionAffinity.CookieSessionAffinity.Name}}  httponly;
         {{ else }}
-        least_conn;
+        # Load balance algorithm; empty for round robin, which is the default
+        {{ $cfg.LoadBalanceAlgorithm }}
```

Comment:
round_robin isn't actually valid here - to get the default behavior we need to leave this line empty, without even a ;